### PR TITLE
Simple metrics

### PR DESCRIPTION
**Pull request by dmanchon**

### CODE REVIEW

**Review by vangheem — CHANGES_REQUESTED** [review]
I just have a question about recording lead time. Please check!

File: kafkaesk/app.py [hunk 1]
```diff
@@ -155,10 +182,40 @@ async def __run(self) -> None:
...
+                    # No error metric
+                    CONSUMED_MESSAGES.labels(
+                        stream_id=record.topic,
+                        error=NOERROR,
+                        partition=record.partition,
+                        group_id=self._subscription.group,
+                    ).inc()
+
+                    # Calculate the time since the message is send until is successfully consumed
+                    lead_time = time.time() - record.timestamp / 1000  # type: ignore
```
Comment (vangheem) [inline kafkaesk/app.py:202]: Shouldn't lead time be recorded before a message is processed?

**Review by vangheem — APPROVED** [review]
Looks great!

**Comment by vangheem** [comment]
0.1.20 released.

File: kafkaesk/metrics.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+NOERROR = "none"
+
+PUBLISHED_MESSAGES = client.Counter(
+    "published_messages", "Number of published messages", ["stream_id", "partition", "error"]
+)
+
+
+CONSUMED_MESSAGES = client.Counter(
+    "consumed_messages",
```
Comment (pfreixes) [inline kafkaesk/metrics.py:11]: I would have prefixed these metrics with the `kafkaesk` suffix for avoiding any collision with other metric names used by the users.

Comment (vangheem) [reply kafkaesk/metrics.py:11]: We can update!

**Pull request merged**